modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -893,22 +841,10 @@ private String _getSystemObjectEntryId(
 		return systemObjectEntryJSONObject.getString("id");
 	}
 
-	private String _invoke(Http.Method httpMethod, String location)
-		throws Exception {
-
-		Http.Options options = _createOptions(httpMethod, location);
-
-		return HttpUtil.URLtoString(options);
-	}
-
-	private int _invokeGetHttpCode(String location) throws Exception {
-		Http.Options options = _createOptions(Http.Method.GET, location);
-
-		HttpUtil.URLtoString(options);
-
-		Http.Response response = options.getResponse();
+	private String _invokeGetHttpCode(String endpoint) throws Exception {
```

Comment:
I think this name is not very descriptive as the http code (int) is not being returned

Reply:
This has not been updated..